plugins/inputs/bcache/bcache.go

```diff
@@ -20,8 +20,44 @@ import (
 var sampleConfig string
 
 type Bcache struct {
-	BcachePath string
-	BcacheDevs []string
+	BcachePath string   `toml:"bcachePath"`
+	BcacheDevs []string `toml:"bcacheDevs"`
```

Comment:
I would prefer
```suggestion
	BcachePath string   `toml:"bcache_path"`
	BcacheDevs []string `toml:"bcache_devs"`
```

Reply:
@srebhan Yes, me too. But I don't want to change names of current properties (look at `sample.conf` and `README`).

Reply:
I'm pretty sure it works both ways. Let's keep your version for now, but check if actually both works and change all the options instances to snake-case in a separate PR. Actually something we could use a linter for:

All exported fields of an exported struct _must_ have a `toml` annotation in snake_case. :-)

Reply:
@srebhan 
After changing toml annotations to:
```
type Bcache struct {
	BcachePath string   `toml:"bcache_path"`
	BcacheDevs []string `toml:"bcache_devs"`
}
```

Telegraf fails to start:
```
2024-10-09T09:16:53Z I! Loading config: telegraf_bcache.conf
2024-10-09T09:16:53Z E! loading config file telegraf_bcache.conf failed: plugin inputs.bcache: line 157: configuration specified the fields ["bcachePath" "bcacheDevs"], but they were not used. This is either a typo or this config option does not exist in this version.
```